tests/integration/institutions/test_create_institution.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any, Callable
+
+from fastapi.testclient import TestClient
+from starlette import status
+
+INSTITUTION_URL = "/institutions"
+
+
+def test_create_institution(
```

Comment:
You should also test what happens if you try to create a new institution with an already existing name and department.